sources-user.js

```diff
@@ -18,6 +18,15 @@ module.exports = [
       patterns: `docs/en/userguide/**`,
     }
   },
+  {
+    resolve: `gatsby-source-git`,
+    options: {
+      name: `user--5--optional_features/elemental`,
+      remote: `https://github.com/silverstripe/silverstripe-elemental`,
+      branch: `5.2`,
+      patterns: `docs/en/userguide/**`,
+    }
+  },
```

Comment:
Just realised this is a duplicate of the `dnadesign/silverstripe-elemental` entry that's already in this file. Please update that entry.

Please also use the `5` branch for user docs. We so rarely update these docs that we can effectively treat the next minor branch as stable.

Reply:
Removed, just use the existing one